src/App.js

```diff
@@ -5,8 +5,9 @@ import Header from './components/Header';
 function App() {
   return (
     <div className="App">
+      <Header/>
       <header className="App-header">
```

Comment:
Thank you for your contribution. Is <header className="App-header"> still needed? I guess only one header should be on a website so this could be deleted.

Reply:
Since you opened a new PR this one can be closed.